packages/app/src/components/Root/Root.tsx

```diff
@@ -106,6 +107,7 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
         <SidebarItem icon={LibraryBooks} to="docs" text="Docs" />
         <SidebarItem icon={LayersIcon} to="explore" text="Explore" />
         <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
+        <SidebarItem icon={List} text="Template tasks" to="create/tasks" />
```

Comment:
Dunno if this ever quite makes sense to add to the sidebar. Perhaps we could make it accessible via the kebab menu on the scaffolder page instead?

Reply:
Yep, think that's a much better idea.